### PR TITLE
CSCFAIRMETA-276: Show message on Metax RPC success.

### PR DESCRIPTION
**Pull request by tahme**

### CODE REVIEW

File: src/views/Editor.vue [hunk 1]
```diff
@@ -678,8 +678,12 @@ export default {
...
 					)
 					await this.openRecord(this.id) // reopen updated dataset so possible old version tag gets updated
-					if (response.data && response.data.new_id && await this.confirmOpenNewVersion()) {
-						this.$router.replace({ name: 'tab', params: { id: response.data.new_id, tab: this.$route.params.tab }})
+					if (response.data && response.data.new_id) {
+						if (await this.confirmOpenNewVersion()) {
+							this.$router.replace({ name: 'tab', params: { id: response.data.new_id, tab: this.$route.params.tab }})
+						}
+					} else {
+						this.$root.showAlert("Cumulative state changed succesfully.", "primary")
```
Comment (jppiiroinen) [inline src/views/Editor.vue]: typo in 'succesfully'

File: src/views/Editor.vue [hunk 2]
```diff
@@ -731,8 +735,12 @@ export default {
...
 				)
 				await this.openRecord(this.id) // reopen updated dataset so possible old version tag gets updated
-				if (response.data && response.data.new_id && await this.confirmOpenNewVersion()) {
-					this.$router.replace({ name: 'tab', params: { id: response.data.new_id, tab: this.$route.params.tab }})
+				if (response.data && response.data.new_id) {
+					if (await this.confirmOpenNewVersion()) {
+						this.$router.replace({ name: 'tab', params: { id: response.data.new_id, tab: this.$route.params.tab }})
+					}
+				} else {
+					this.$root.showAlert("Folder content refreshed succesfully.", "primary")
```
Comment (jppiiroinen) [inline src/views/Editor.vue]: same typo here

**Review by jppiiroinen — CHANGES_REQUESTED** [review]
a couple of typos

File: src/views/Editor.vue (hunk 1 above)
Comment (tahme) [reply src/views/Editor.vue]: true. fixed now.

File: src/views/Editor.vue (hunk 2 above)
Comment (tahme) [reply src/views/Editor.vue]: fixed

**Review by jppiiroinen — APPROVED** [review]
ok

**Pull request merged**